extensions/wikia/AdminDashboard/css/AdminDashboard.scss

```diff
@@ -528,10 +529,12 @@ $color-highlight-column: rgba($color-page, 0);
 	}
 
 	.mw-special-AdminDashboard .WikiaMainContentContainer {
-		margin-right: $admin-dashboard-rail-width;
+		@if ( $responsive-layout-enabled ) {
+			margin-right: $admin-dashboard-rail-width;
+		}
 		padding: 0;
 
-		@media screen and (max-width: $breakpoint-small) {
+		@media screen and (max-width: 1023px) {
```

Comment:
It's same value in responsive and breakpoints, but `$breakpoint-small` variable is unavailable in breakpoints - instead of importing two different layouts I've done this quick fix. All "fixes" will be reviewed in later ticket.